packages/strapi-admin/files/public/app/containers/LocaleToggle/index.js

```diff
@@ -8,19 +8,19 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { selectLocale } from '../LanguageProvider/selectors';
 import { changeLocale } from '../LanguageProvider/actions';
-import { appLocales } from '../../i18n';
+import { languages } from '../../i18n';
 import { createSelector } from 'reselect';
 import styles from './styles.scss';
 import Toggle from 'components/Toggle';
 
 export class LocaleToggle extends React.Component { // eslint-disable-line
   render() {
     const messages = {};
-    appLocales.forEach(locale => { messages[locale] = locale.toUpperCase(); });
+    languages.forEach(locale => { messages[locale] = locale.toUpperCase(); });
```

Comment:
Using `reduce`could be better to avoid the use of the `forEach` method to populate a previously declared variable.

```js
const messages = languages.reduce((acc, locale) => (acc[locale] = locale.toUpperCase(), acc), {});
```